my-app/src/components/episodes/EpisodeCard.js

```diff
@@ -0,0 +1,11 @@
+const EpisodeCard = (props) => {
+  const episode = props.episode
```

Comment:
This can be simplified with destructering like this:

```suggestion
const EpisodeCard = ({ episode }) => {
```